job_groups/opensuse_tumbleweed.yaml

```
@@ -294,7 +294,7 @@ scenarios:
           settings:
             CONTAINER_RUNTIMES: 'podman'
             K3S_INSTALL_UPSTREAM: '1'
-      - containers_host_podman_bats:
+      - containers_host_podman_integration:
```

```suggestion
      - containers_host_podman_testuite:
```
Petition to rename it simply to `podman_testsuite`. WDYT?

---

Done